feat: MVC 대공사

dlwlstks96 opened the pull request:
@planepaper 
 - 함수와 변수의 시작 글자 각각 대문자와 소문자로 수정
 - 클래스 내 멤버 함수 및 변수를 public 먼저 선언

dlwlstks96 committed:
feat: 시작 글자를 함수는 대문자로, 변수는 소문자로 수정

 - 함수와 변수의 시작 글자 각각 대문자와 소문자로 수정
 - 클래스 내 멤버 함수 및 변수를 public 먼저 선언

dlwlstks96 committed:
feat: 입력 받는 creditLevel에 따라 호출되는 switch문 수정

- 보다 간결한 코드로 수정

dlwlstks96 commented:
@planepaper

- 입력 받는 creditLevel에 따라 호출되는 switch문 보다 간결하게 수정

dlwlstks96 committed:
feat: 입금 및 출금 시 호출되는 함수 구조 정리

- 입금, 출금 각 Menu 함수에서 정보를 입력 받고 print까지 처리
- 그 외 Deposit, Withdraw 등의 함수에선 각자의 기능만 구현하도록 수정

dlwlstks96 commented:
@planepaper 

feat: 입금 및 출금 시 호출되는 함수 구조 정리

- 입금, 출금 각 Menu 함수에서 정보를 입력 받고 print까지 처리
- 그 외 Deposit, Withdraw 등의 함수에선 각자의 기능만 구현하도록 수정

※ 아직 MVC로 만드는 과정 시작도 안됐지만 그 단계까지 도달하려고 편의상 PR 제목을 그렇게 했어~

dlwlstks96 committed:
feat: 뱅킹 시스템의 메뉴를 별도의 함수가 아닌 main에서 구현

- AccountHandler의 멤버 함수였던 menu 삭제
- 메뉴의 4,5번 기능을 별도의 함수로 구현(ShowAccount, DeleteAcc)

dlwlstks96 commented:
@planepaper 

feat: 뱅킹 시스템의 메뉴를 별도의 함수가 아닌 main에서 구현

- AccountHandler의 멤버 함수였던 menu 삭제
- 메뉴의 4,5번 기능을 별도의 함수로 구현(ShowAccount, DeleteAcc)

planepaper reviewed (commented):
커밋 내용이 기능을 추가한게 아니라, 코드를 수정한거니까
feat: 말고, refactor: 라고 썼으면 더 좋았을거같네~

그리고 여기 커밋 내용에 project1.cpp 파일을 제외한내용이 있는데, 세부 메세지에 그 내용도 썼으면 좋았을거같아~

planepaper reviewed (commented):
커밋 내용이 기능을 추가한게 아니라, 코드를 수정한거면
feat: 말고, refactor: 라고 썼으면 더 좋았을거같네~

그리고 여기 커밋 내용에 project1.cpp 파일을 제외한내용이 있는데, 세부 메세지에 그 내용도 썼으면 좋았을거같아~

planepaper commented:
그리고 커밋 내용 굳이 복사해서 코멘트로 다시 안남겨도돼 ㅎㅎ 내가 알아서 잘 볼 수 있어! ㅋㅋ

planepaper commented:
그리고 이거 작업이 계속 진행중인거야 아니면 내가 머지해줬으면 하는 건걸까??
이 브렌치에다가 계속 작업을 할건데 그냥 PR만 미리 올려놓은거면
https://github.blog/2019-02-14-introducing-draft-pull-requests/
PR을 생성할 때 draft로 생성하면 돼 ㅎㅎ

이 PR을 작업중인걸로 바꾸고싶으면,, 설정눌러서 draft로 변경하기가 있을거야 한번 찾아봐 ㅎㅎ 내가 올린PR이 아니라서 어떻게 바꾸는지는 모르겠네 ㅋㅋ draft로 바꾸게되면 PR은 open되어있는데 내가 merge버튼을 누를 수 없게될거야! ㅎㅎ

dlwlstks96 commented:
코멘트 감사합니다~ 마지막 챕터 프로젝트까지 여기에 같이 작업하기 위해
draft로 변경하였습니다!

dlwlstks96 committed:
feat: 배열 클래스의 선언과 정의 & 대입 연산자

- 경계 검사를 수행하는 AccountArray 배열 클래스 생성
- Account 클래스의 대입 연산자를 깊은 복사가 진행되도록 정의
※ 누락된 AccountArray 헤더, 소스 파일 추가

dlwlstks96 commented:
@planepaper 
마지막 프로젝트 수행하여 커밋하였습니다~:)
draft 풀도록 하겠습니다

planepaper commented:
@dlwlstks96 
진산아~
너가 git add -p만 해서 새롭게 작성한 AccountArray.h랑 AccountArray.cpp은 커밋에 누락된거같아~
위 파일들 git add 하고 git commit --amend 이용해서 다시 커밋 해줬으면 좋겠다~

항상 git status로 모든 파일들이 제대로 올라왔는지 커밋하기전에 한번 더 확인해보면 좋겠네~

dlwlstks96 commented:
미안 확인했어야하는데ㅠㅠ
누락되었던 두 파일 추가해서 올렸어!!

dlwlstks96 committed:
test1

dlwlstks96 committed:
test2
